js/classes/entities/midground/enemies/village/snake.js

```diff
@@ -109,7 +109,7 @@ class Snake {
                 if (secondsSinceLastAttack > Snake.ATTACK_COOLDOWN) {
                     // if it's been long enough, start a new attack 
                     this.state = "pursue";
-                    this.base.setTargetX(CHAD.pos.x);
+                    this.base.setTargetX(CHAD.pos.x + CHAD.scaleBoundingBoxOffset().x);
```

Comment:
Do we need this line? It appears we already do this in enemyBase.js

Reply:
Good question. It is necessary because the target is not set in `EnemyBase` if the enemy is attacking, which it would be in this case. This causes the snake to skip its first attack.